### PR TITLE
refactor(iot-device): Fix documentation comments

### PR DESCRIPTION
**Pull request by abhipsaMisra**

### CODE REVIEW

File: iothub/device/src/ClientOptions.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ namespace Microsoft.Azure.Devices.Client
     public class ClientOptions
     {
         /// <summary>
-        /// The Digital Twin Model Id associated with the device client instance.
+        /// The Digital Twins model Id associated with the device client instance.
```
Comment (drwill-ms) [inline iothub/device/src/ClientOptions.cs:12]: you can lower case digital twins too, per your other inquiry

Comment (vinagesh) [inline iothub/device/src/ClientOptions.cs:12]: do you need a plural here - twins?

Comment (abhipsaMisra) [reply iothub/device/src/ClientOptions.cs:12]: ok

Comment (abhipsaMisra) [reply iothub/device/src/ClientOptions.cs:12]: yes, that is the term that we are using when referring to this feature/ service.

**Pull request merged**